deepchem/models/torch_models/layers.py

```diff
@@ -4942,15 +4942,15 @@ def __init__(self, n_one: List[int], n_two: List[int], no_of_atoms: int,
         # Initializing the first layer (first layer has different dims than others)
         self.v.append(
             nn.Linear(8 + 3 * 4 * self.no_of_atoms, self.n_one[0], bias=True))
-        #filling the weights with 2.5e-7 for faster convergence
-        self.v[0].weight.data.fill_(2.5e-7)
-        self.v[0].bias.data.fill_(2.5e-7)
+        #filling the weights with 1e-3 for faster convergence
+        self.v[0].weight.data.fill_(1e-3)
+        self.v[0].bias.data.fill_(1e-3)
         self.v[0].weight.data = self.v[0].weight.data
         self.v[0].bias.data = self.v[0].bias.data
 
         self.w.append(nn.Linear(4, self.n_two[0], bias=True))
-        self.w[0].weight.data.fill_(2.5e-7)
-        self.w[0].bias.data.fill_(2.5e-7)
+        self.w[0].weight.data.fill_(1e-3)
```

Comment:
Right now fixing the initial weight to 1e-3, which gives me decent results for H2. In the Jax implementation of ferminet, they used a different initialization of weights. I will look into it and put it up all these in the last 'optimization' PR(better learning rates, better initialization width,etc)